[2025-07-15T04:00:01.854+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_analise_desempenho_revendas.executar_query_gnio scheduled__2025-07-14T04:00:00+00:00 [queued]>
[2025-07-15T04:00:01.870+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_analise_desempenho_revendas.executar_query_gnio scheduled__2025-07-14T04:00:00+00:00 [queued]>
[2025-07-15T04:00:01.870+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-07-15T04:00:01.951+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): executar_query_gnio> on 2025-07-14 04:00:00+00:00
[2025-07-15T04:00:01.963+0000] {standard_task_runner.py:60} INFO - Started process 1258686 to run task
[2025-07-15T04:00:01.969+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'DAG_analise_desempenho_revendas', 'executar_query_gnio', 'scheduled__2025-07-14T04:00:00+00:00', '--job-id', '525', '--raw', '--subdir', 'DAGS_FOLDER/DagAnaliseDesempenhoRevendas.py', '--cfg-path', '/tmp/tmplu35m_ff']
[2025-07-15T04:00:01.973+0000] {standard_task_runner.py:88} INFO - Job 525: Subtask executar_query_gnio
[2025-07-15T04:00:02.389+0000] {task_command.py:423} INFO - Running <TaskInstance: DAG_analise_desempenho_revendas.executar_query_gnio scheduled__2025-07-14T04:00:00+00:00 [running]> on host 99f5714845f7
[2025-07-15T04:00:02.621+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_analise_desempenho_revendas' AIRFLOW_CTX_TASK_ID='executar_query_gnio' AIRFLOW_CTX_EXECUTION_DATE='2025-07-14T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-14T04:00:00+00:00'
[2025-07-15T04:00:02.651+0000] {base.py:83} INFO - Using connection ID 'gnio' for task execution.
[2025-07-15T04:00:03.654+0000] {warnings.py:109} WARNING - /opt/***/dags/DAG_atualizacao_desempenho_revendas/AnaliseDesempenhoRevendas.py:16: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(sql, connection_database)

[2025-07-15T04:00:05.666+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-15T04:00:05.687+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=DAG_analise_desempenho_revendas, task_id=executar_query_gnio, execution_date=20250714T040000, start_date=20250715T040001, end_date=20250715T040005
[2025-07-15T04:00:05.836+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-15T04:00:06.135+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
